### PR TITLE
feature/game entities

### PR DESCRIPTION
**Pull request by MartinPELCAT**
- add base game entities
- stop pushing empty classes
- add routes, not implemented
- change DTO decorators

### CODE REVIEW

File: src/main/java/com/ynov/master/mobile/game/medieval/warfare/controller/GameController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    @Autowired
+    private GameService gameService;
+
+
+    @PostMapping("/join")
+    @ApiOperation(value = "Join a new game")
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "Something went wrong"),
+            @ApiResponse(code = 422, message = "Invalid username/password supplied")})
```
Comment (BriceMichalski) [inline src/main/java/com/ynov/master/mobile/game/medieval/warfare/controller/GameController.java]: Mauvaise doc

File: src/main/java/com/ynov/master/mobile/game/medieval/warfare/model/PlayerState.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import lombok.Data;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+import org.bson.types.ObjectId;
+
+@Data
+public class PlayerState {
+  @BsonProperty("_id")
+  private ObjectId id;
+
+  Position position;
```
Comment (BriceMichalski) [inline src/main/java/com/ynov/master/mobile/game/medieval/warfare/model/PlayerState.java]: pas de BsonPropetu la dessus ?

File: src/main/java/com/ynov/master/mobile/game/medieval/warfare/controller/GameController.java (hunk 1 above)
Comment (MartinPELCAT) [reply src/main/java/com/ynov/master/mobile/game/medieval/warfare/controller/GameController.java]: ✔️

File: src/main/java/com/ynov/master/mobile/game/medieval/warfare/model/PlayerState.java (hunk 2 above)
Comment (MartinPELCAT) [reply src/main/java/com/ynov/master/mobile/game/medieval/warfare/model/PlayerState.java]: ✔️

**Review by BriceMichalski — APPROVED** [review]

**Pull request merged**